plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -198,6 +198,9 @@ func (s *Plugin) Validate(ctx context.Context, a admission.Attributes, o admissi
 					if projSource.ServiceAccountToken != nil {
 						return admission.NewForbidden(a, fmt.Errorf("a mirror pod may not use ServiceAccountToken volume projections"))
 					}
+					if projSource.ClusterTrustBundle != nil {
```

Comment:
@liggitt can you copy this to where kubelet prevents itself from reading API sources.

Reply:
drop this check from this admission plugin, it's not really related to service accounts